code/modules/uplink/uplink_items/explosive.dm

```diff
@@ -62,6 +62,11 @@
 	item = /obj/item/storage/box/syndie_kit/emp
 	cost = 2
 
+/datum/uplink_item/explosives/emp/New()
+	..()
+	if(HAS_TRAIT(SSstation, STATION_TRAIT_CYBERNETIC_REVOLUTION))
+		cost *= 3
```

Comment:
feeble copper wired crew's single saving grace

Reply:
Won't stop lings from spamming it though